src/app/books/itemNewAll/page.tsx

```diff
@@ -1,5 +1,39 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+
+import { getBooksByGroupNameRequest, resetBooks } from '@/app/actions/types';
+import { RootState } from '@/app/reducers';
+import { Container, Button } from '@mui/material';
+import { useDispatch, useSelector } from 'react-redux';
+
+import BooksContainerForGroupName from '../../components/Book/BooksContainerForGroupName';
+import { AppDispatch } from '../../store/store';
+
 const ItemNewAll = () => {
-  return <h1>여기에 itemNewAll이 보여집니다.</h1>;
+  const dispatch = useDispatch<AppDispatch>();
+  const { books } = useSelector((store: RootState) => store.book);
+  const [page, setPage] = useState<number>(1);
+
+  useEffect(() => {
+    dispatch(resetBooks());
+    dispatch(getBooksByGroupNameRequest('ItemNewAll', page, 20));
```

Comment:
I think you could also extract the GroupName via useRouter, otherwise it is not so dynamic here.